### DISCUSSION TITLE
what backend for a github display

### DISCUSSION THREAD

**Discussion by PBrockmann** [pyvista/pyvista#3637] [Q&A] [answered]
Hi,
I am a little confused in what backend I should use to have a notebook properly displayed with pyvista objects from a github repository.
I have used the panel backend that seems appropriate.

I have a correct display with 
https://htmlpreview.github.io/?https://github.com/PBrockmann/LSCE_notebooks/blob/main/test.html

Is is correct ?
Why the html view of a notebook does not display what htmlpreview does directly ?

**Comment by banesullivan** [comment, marked as answer]
You may need to consult the people working on panel but AFAIK, GitHub will not display dynamic HTML from Notebooks, and that's the whole value proposition of websites like `nbviewer`: https://nbviewer.org/

**Reply by banesullivan** [reply]
AFAIK, it's the same reason you can't have dynamic HTML or iframes in Markdown previews n GitHub

**Reply by banesullivan** [reply]
Take a look at GitHub's Spec: "6.11 Disallowed Raw HTML (extension)" https://github.github.com/gfm/#example-657

> GFM enables the tagfilter extension, where the following HTML tags will be filtered when rendering HTML output:
> <title> <textarea> <style> <xmp> <iframe> <noembed> <noframes> <script> <plaintext>

Notably, the `<script>` tag is disallowed on GitHub yet is absolutely required for rendering the scenes
